script/Ciel/Day 13/13_01_01_CIEL13_1_2.txt

```diff
@@ -36,43 +37,49 @@ Enough of that for now.
 [sha:95083bf0cf5e8dc9a536105a07134a5723937d0d]{
 -- Page 67, Offset 33320.
 -- 全身に巣くった痛みを吐き出さないと、<正常|マトモ>な思考もおぼつかない。
-If I don't expel the pain that's plaguing my entire body, I won't even be able to think straight.
+I won't be able to think straight unless I rid myself of this pain that's burrowed into every corner of my body.
 }
 [sha:8cef1386f07c4a841a7a54d9bffbe7de9e8f52d4]{
 -- Page 68, Offset 33321.
 -- 「……はあ……はあ……はあ……」
-"...Hah... hah... hah..."
+"%{i}Hah%{/i}... %{i}hah%{/i}... %{i}hah%{/i}..."
 }
 [sha:834d8fbd25cf91773b244bc1ac841100d3f7e1eb]{
 -- Page 68, Offset 33322.
 -- 息があがっている。
-My breathing grows rough.
+I start breathing heavily.
 }
 [sha:bc621cbd0d0824f6b2556f2a7158c6029c6e7c94]{
 -- Page 68, Offset 33323.
 -- 早く。
-Faster.
+Hurry up.
 }
 [sha:15979d0e03a25414b8cc559c065cc2e6469a9f19]{
 -- Page 68, Offset 33324.
 -- この、芽生えたばかりの<志|ワ><向|タ><性|シ>を外にはき出さないと、中身の<自|お><分|れ>が消されてしまう。
 // i swear nasu is writing random bullshit what the fuck is this
-If I don't get rid of this newly-sprouted <intentionality|m e>, then the me on the inside will be erased.
+// makes total sense: Roa is a will (intention)
+If I don't get rid of this new <intentionality|%{i}m e%{/i}   > that's sprouted within me,
+ then my own inner self will be erased.
```

Comment:
To go with the previous (since はき出さない gets reused here):

>If I don't get spew out this <directive|%{i}m e%{/i}   > that's sprouted within me,
> then my own inner self will be erased.

Also, I think Nasu has mispelled 指向性 here.